XMonad/Hooks/StatusBar.hs

```diff
@@ -323,6 +323,19 @@ statusBarPipe cmd xpp  = do
 -- > xmobar1      = statusBarPropTo "_XMONAD_LOG_3" "xmobar -x 1 ~/.config/xmobar/xmobarrc1"       (pure pp1)
 -- >
 -- > main = xmonad $ withSB (xmobarTop <> xmobarBottom <> xmobar1) myConfig
+-- >
+-- And here is an example of the related xmobar configuration for the multiple 
+-- status bars mentioned above:
+-- > xmobarrc_top
+-- > Config {
+-- > ...
+-- > , Run XPropertyLog "_XMONAD_LOG_1"  ]
+-- > , sepChar = "%"
+-- > , alignSep = "}{"
+-- > , template = "%_XMONAD_LOG_0% }{...}
+-- >
+-- @XPropertyLog@ can be replace by @UnsafeXPropertyLog@ if you want to pass 
+-- <action=...> to it.
```

Comment:
This feels like it would make people try and manually pass `<action=…>` to xmobar... Not sure though, maybe it's fine

Reply:
I think it is kind of OK. Not sure. Maybe I should delete it?

Reply:
We don't mention `UnsafeStdinReader` here, so maybe it's best if we leave it to the modules that actually need it to mention it. `X.U.ClickableWorkspaces` does that, so I think we can drop this here.

Reply:
Thanks, I have removed it.